fullstack-cert/react-projects/usememo-currency-converter/index.jsx

```diff
@@ -0,0 +1,35 @@
+export default function CurrencyConverter() {
```

Comment:
we will need to include this import for them.

```suggestion
const { useMemo } = React;

export default function CurrencyConverter() {
```